designer/server/src/plugins/routes/api.test.js

```diff
@@ -112,7 +112,8 @@ describe('Server API', () => {
     )
 
     expect(result.statusCode).toBe(500)
-    expect(result.result?.err.message).toMatch('Schema validation failed')
+    console.log(result.result)
+    expect(result.result.err.message).toMatch('Schema validation failed')
```

Comment:
@davidjamesstone This PR is amazing 🙌

Probably just these two lines left in by accident?